tests/utest/test_cli.py

```diff
@@ -60,19 +58,6 @@ def test_configure_not_existing_transformer(self, name, similar):
         result = run_tidy(args, exit_code=1)
         assert expected_output == result.output
 
-    def test_transformer_order(self):
```

Comment:
Moved some of the tests from test_cli to test_load_transformers since those were not testing cli but loading transformers.